src/python/pants/backend/codegen/grpcio/python/grpcio_run.py

```diff
@@ -45,6 +48,11 @@ def execute_codegen(self, target, target_workdir):
       if exit_code != 0:
         raise TaskError('{} ... exited non-zero ({}).'.format(cmdline, exit_code),
                         exit_code=exit_code)
+      # Create __init__.py in each subdirectory of the target
+      # directory so that setup_py recognizes them as modules.
```

Comment:
Nit: we use a 100 character line limit in Pants, so this might be able to be condensed into one line. Even if you can't fit it all on one, recommended to fit up to 100 characters on the first line and then the rest on the second for consistency with the rest of the codebase.